app/gui/src/model/execution_context/plain.rs

```diff
@@ -61,6 +62,8 @@ pub struct ExecutionContext {
     pub is_ready: crate::sync::Synchronized<bool>,
     /// Component groups defined in libraries imported into the execution context.
     pub component_groups: RefCell<Rc<Vec<ComponentGroup>>>,
+    /// Execution environment of the context.
+    pub execution_environment: Rc<Cell<ExecutionEnvironment>>,
```

Comment:
Does it need to be in `Rc`? This model does not implement `CloneRef`, and other fields are just `RefCell`s without being wrapped in `Rc`.

Reply:
Good catch.